### ISSUE TITLE
missing shebang in .travis.install.sh

### ISSUE THREAD

**Issue by FunTimeCoding** [doctrine/instantiator#17]
ShellCheck brought this up.

```
In ./vendor/doctrine/instantiator/.travis.install.sh line 1:
set -x
^-- SC2148: Tips depend on target shell and yours is unknown. Add a shebang.
```

**Comment by Ocramius** [comment]
@FunTimeCoding send a PR ;-)

**Comment by Majkl578** [comment]
This has been fixed ages ago in  #18.
